content/kubernetes/active-active/preview/edit-rerc.md

```diff
@@ -0,0 +1,88 @@
+---
+Title: Edit Redis Enterprise remote clusters
+linkTitle: Edit RERC
+description: Edit the configuration details of an existing RERC with Redis Enterprise for Kubernetes.
+weight: 60
+alwaysopen: false
+categories: ["Platforms"]
+aliases: {
+    /kubernetes/active-active/preview/edit-rerc
+
+}
+---
+{{<banner-article bannerColor="#fff8dc">}}
+This feature is currently in public preview. Contact Redis support if you plan to use this feature.
+{{</banner-article>}}
+
+Before a RedisEnterpriseCluster (REC) can participate in an Active-Active database, it needs an accompanying RedisEnterpriseRemoteCluster (RERC) custom resource. The RERC contains details allowing the REC to link to the RedisEnterpriseActiveActiveDatabase (REAADB). The RERC resource is listed in the REAADB resource to become a participating cluster for the Active-Active database.
+
+For more details, see the [RERC API reference](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_remote_cluster_api.md).
+
+## Edit RERC
+
+Use the `kubectl patch rerc <rerc-name> --type merge --patch` command to patch the local RERC custom resource with your changes. For a full list of available fields, see the [RERC API reference](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_remote_cluster_api.md).
+
+The following example edits the `dbFqdnSuffix` field for the RERC named `rerc1`.
+
+```sh
+kubectl patch rerc rerc1 --type merge --patch \
+'{"spec":{"dbFqdnSuffix": "-example2-cluster-rec1-ns1.redis.com"}}'
+```
+
+## Update RERC secret
```

Comment:
The numbered steps in this section are not being rendered correctly. Adding tabs/spaces before the non-numbered sentences and code snippets should fix this issue.